web/src/screens/settings/Account.tsx

```diff
@@ -160,6 +161,22 @@ function Credentials() {
 }
 
 function OIDCAccount() {
+  const auth = AuthContext.get();
+  const [imageError, setImageError] = useState(false);
```

Comment:
Same state comment here, not sure what this adds versus checking if the variable exists and is not empty.